server/v2/appmanager/circuit.go

```diff
@@ -1,9 +0,0 @@
-package appmanager
-
-import "context"
-
-// CircuitBreaker is an interface that defines the methods for a circuit breaker.
-// TODO replace with pre message hooks
-type CircuitBreaker interface {
```

Comment:
we should leave this for now in order to not forget we need it. also we will need it for backwards compatability